jobs/build-arch.Jenkinsfile

```diff
@@ -20,11 +20,8 @@ node {
         echo "Running in developer mode on ${env.JENKINS_URL}."
     }
 
-    developer_prefix = pipeutils.get_config('developer-prefix')
-    src_config_url = pipeutils.get_config('source-config-url')
-    src_config_ref = pipeutils.get_config('source-config-ref')
-    s3_bucket = pipeutils.get_config('s3-bucket')
-    gcp_gs_bucket = pipeutils.get_config('gcp-gs-bucket')
+    pipecfg = pipeutils.load_config()
+    developer_prefix = pipecfg['developer-prefix']
```

Comment:
Any chance we could use `pipecfg`, but still declare/define these variables up front? i.e. we'd add a new var for `notify_slack` and here we would `notify_slack = pipecfg['notify_slack']`.

I think it makes later parts of the code easier to read if you don't worry about where the definition of the variable comes from, but just what the variable means. The extra `pipecfg[]` in each use now makes you think about that.